strapi/design-system
# [Dark mode] Memorize mode

## What

- Fixed mode chosen not memorized
- Refacto Theme component used in `preview.js`
- Added `window.matchMedia` to `test-bundler.js `  ➡️
- Needed to use `import { useDarkMode } from 'storybook-dark-mode'` in Theme component

## How to test

- Launch Storybook
- Change mode 
- Switch to a different story, reload the page
- It should memorize your mode chosen
- Shouldn't break dark mode e2e tests
